app/services/webhook_service.py

```diff
@@ -7,12 +7,20 @@
 from app.schemas.webhook_schemas import WebhookRequestData, AccountConnectedEvent, ContentEvent, \
     ContentGroupEvent, ProfileEvent, TransactionEvent, PayoutEvent, BalanceEvent, ActivityArtistEvent, \
     ActivityContentEvent, ProfileAudienceEvent, ContentCommentEvent, PublishContentEvent, ProfileAnalyticsEvent, \
-    ContentsFetchEvent, AudienceOverlapEvent
+    ContentsFetchEvent, ProfessionalContentsFetchEvent, AudienceOverlapEvent
 from app.services.resource_service import fetch_contents_by_ids, fetch_account_by_id, fetch_content_groups_by_ids, \
     fetch_profile_by_id, fetch_social_transactions_by_ids, fetch_social_payouts_by_ids, fetch_balances_by_ids, \
     fetch_activity_artists_by_ids, fetch_activity_contents_by_ids, fetch_profile_audience_by_account_id, \
     fetch_content_comments_by_content_id_account_id, fetch_commerce_transactions_by_ids, fetch_commerce_payouts_by_ids, \
-    fetch_publish_content_by_id, fetch_profile_analytics_by_id, fetch_async_contents_by_id, fetch_audience_overlap_by_id
+    fetch_publish_content_by_id, fetch_profile_analytics_by_id, fetch_async_contents_by_id, fetch_audience_overlap_by_id, \
+    fetch_async_professional_contents_by_id
+
+# whitelisted work_platform_ids for professional/contents
+PROFESSIONAL_WORK_PLATFORM_IDS = ["36410629-f907-43ba-aa0d-434ca9c0501a"]
```

Comment:
* Move this to `app/utils/constants.py`
* Create a constant for `36410629-f907-43ba-aa0d-434ca9c0501a` this WP, then add that constant to the list
* Since IDs are not easily understood by humans, it requires an extra step to determine which work platform an ID belongs to